models/prices/aptos/prices_aptos_tokens.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+        schema='prices_aptos',
+        alias = alias('tokens'),
```

Comment:
```suggestion
        alias = 'tokens',
```

Reply:
My bad. Thank you @Hosuke for the help 🙏

Reply:
For APT contract address, should I change
`cast(contract_address as varchar) as contract_address` to `from_hex(contract_address) as contract_address

Reply:
It's not hexstring, let's try `from_base58` first.